docs/provisioning-methods-and-credentialszip.adoc

```diff
@@ -0,0 +1,63 @@
+// Copy of this topic: https://docs.ota.here.com/concepts/provisioning-methods-and-credentialszip.html adapted for aktualizr repo. Replaces duplicate "credentials.adoc".
+
+= Provisioning methods and credentials.zip
+
+If you're trying to work on integrating {product-name} into your device build, it may be helpful to have a little bit of reference information on how the different provisioning methods work together, and what exactly the magic sauce inside `credentials.zip` is.
+
+== `credentials.zip` file format
+
+First, a table:
+
+The following files are present in credentials.zip:
+
+// tag::credentials-table[]
+
+[options="header"]
+|======================
+| Filename in zip | Purpose | Used by
+| treehub.json | Location and authentications for treehub and Uptane repo | garage-sign, garage-push
+| client.crt | Certificate for TLS client authentication | garage-push
+| client.key | Private key for TLS client authentication | garage-push
+| root.crt | Root CA for TLS client authentication | garage-push
+| autoprov_credentials.p12 | TLS client credentials that are required when provisioning devices with shared credentials | aktualizr, aktualizr-cert-provider
+| autoprov.url | URL for provisioning server | aktualizr, aktualizr-cert-provider
+| root.json | Initial Uptane root.json (for secure bootstrapping) | garage-sign
+| targets.pub | Public key for offline Uptane image signing | garage-sign
+| targets.sec | Private key for offline Uptane image signing | garage-sign
+| tufrepo.url | URL to Uptane repository | garage-sign
+|======================
+
+// end::credentials-table[]
+
+As you can see, the relevant files for the device itself are `autoprov_credentials.p12` and `autoprov.url`.
+
+When you when you link:./enable-implicit-provisioning.html#enable-implicit-provisioning-with-a-hardware-security-module-hsm[provision with Device Credentials via an HSM], the `implicit_writer` function in meta-updater takes over. The initial credentials on the device won't be valid; this is why you need to copy in the generated ones after booting it.
+
+== Configuration options for Provisioning with Device Credentials
```

Comment:
Why don't we specify the same information for shared credential provisioning? It used to be specified in the old docs that are being deleted. May or may not be relevant, but it's a useful reference, even if only for developers.

Reply:
added equivalent info back in.